.gitignore

```diff
@@ -38,3 +38,7 @@ org.eclipse.buildship.core.prefs
 .classpath
 .project
 bin/
+
+# napt triggers
+# https://github.com/sergei-lapin/napt
+NaptTrigger.java
```

Comment:
We can put generated `NaptTrigger.java` files into proper packages, and [disabling NaptTrigger generation](https://github.com/sergei-lapin/napt#disabling-napttrigger-generation).

Reply:
We can't disable the triggers as we don't have any Java sources. The triggers ensure that napt runs on Kotlin-only projects.

Reply:
I meant we can move the triggers into java sourceSets and disable `generateNaptTrigger`. 

<img width="771" alt="image" src="https://user-images.githubusercontent.com/10363352/192253000-3dc275b6-8fa1-40fd-8f53-1784d5d6bda5.png">
<img width="786" alt="image" src="https://user-images.githubusercontent.com/10363352/192253124-314f2c09-b361-4cc3-aa6a-e2cff7c16454.png">

Reply:
Yeah we could, but I quite like that they're not polluting the 'actual' package we use. As they're generated code, it doesn't really matter imo. WDYT?

Reply:
No problem.